### PR TITLE
Fix a bunch of spelling mistakes

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/memgraph/actions/runs/5269880555#summary-14263813140

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/memgraph/actions/runs/5269883621#summary-14263822482

[master < Epic] PR
- [ ] Check, and update documentation if necessary
- [ ] Write E2E tests
- [ ] Compare the [benchmarking results](https://bench-graph.memgraph.com/) between the master branch and the Epic branch
- [ ] Provide the full content or a guide for the final git message

[master < Task] PR
- [ ] Check, and update documentation if necessary
- [ ] Provide the full content or a guide for the final git message

To keep docs changelog up to date, one more thing to do:
- [ ] Write a release note here, including added/changed clauses
- [ ] Tag someone from docs team in the comments

### CODE REVIEW

File: tests/e2e/triggers/CMakeLists.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@ target_link_libraries(memgraph__e2e__triggers__on_update memgraph__e2e__triggers
 add_executable(memgraph__e2e__triggers__on_delete on_delete_triggers.cpp)
 target_link_libraries(memgraph__e2e__triggers__on_delete memgraph__e2e__triggers_common)
 
-add_executable(memgraph__e2e__triggers__privileges privilige_check.cpp)
+add_executable(memgraph__e2e__triggers__privileges privilege_check.cpp)
```
Comment (jsoref) [inline tests/e2e/triggers/CMakeLists.txt:15]: This requires a rename which is currently missing

File: tests/unit/query_semantic.cpp [hunk 2]
```diff
@@ -1002,7 +1002,7 @@ TEST_F(TestSymbolGenerator, MatchUnionReturnSymbols) {
   EXPECT_EQ(symbol_table.max_position(), 8);
 }
 
-TEST_F(TestSymbolGenerator, MatchUnionParameterNameThrowSemanticExpcetion) {
+TEST_F(TestSymbolGenerator, MatchUnionParameterNameThrowSemanticException) {
```
Comment (jsoref) [inline tests/unit/query_semantic.cpp:1005]:
```diff
-Expcetion
+Exception
```

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine.

File: src/query/interpreter.cpp [hunk 3]
```diff
@@ -89,7 +89,7 @@ extern const Event TriggersCreated;
 
 extern const Event QueryExecutionLatency_us;
 
-extern const Event CommitedTransactions;
+extern const Event CommittedTransactions;
```
Comment (gitbuda) [inline src/query/interpreter.cpp:92]: This might be tricky, double check

**Comment by gitbuda** [comment]
@jsoref This changed a lot because we haven't merged it immediately + it has some questionable changes which have to be double-checked. It would be much easier if we do these changes in batches (let's say 10-20 at a time, but without creating immediately 10+ PRs).

I'll close this PR because it's not a priority + it's too bit + we are risking stuff not working.

**Comment by gitbuda** [comment]
Not sure what's the relation, but 10-20 files seem manageable (an assumption is that a signal file won't have a gazillion of changes, which is I think true 99.99% of time)

**Pull request closed without merging**